core/shared/src/main/scala/zio/Fiber.scala

```diff
@@ -858,7 +858,8 @@ object Fiber extends FiberPlatformSpecific {
 
   /**
    * Retrieves the fiber currently executing on this thread, if any. This will
-   * always be `None` unless called from within an executing effect.
+   * always be `None` unless called from within an executing effect and the
+   * runtime executing this effect is configured to enable this feature.
```

Comment:
Me, personally, would appreciate more detailed help possibly with link to corresponding runtime flag :)